docs/sources/enterprise/reporting.md

```diff
@@ -6,47 +6,46 @@ type = "docs"
 aliases = ["/docs/grafana/latest/administration/reports"]
 [menu.docs]
 parent = "enterprise"
-weight = 8
+weight = 400
 +++
 
 # Reporting
 
-> Only available in Grafana Enterprise v6.4+.
+Reporting allows you to automatically generate PDFs from any of your dashboards and have Grafana email them to interested parties on a schedule.
 
-Reporting allows you to generate PDFs from any of your Dashboards and have them sent out to interested parties on a schedule.
+> Only available in Grafana Enterprise v6.4+.
 
 {{< docs-imagebox img="/img/docs/enterprise/reports_list.png" max-width="500px" class="docs-image--no-shadow" >}}
 
-## Dashboard as a Report
-
-With Reports there are a few things to keep in mind, most importantly, any changes you make to the Dashboard used in a report will be reflected in the report. If you change the time range in the Dashboard the time range will be the same in the report as well.
+Any changes you make to a dashboard used in a report are reflected the next time the report is sent. For example, if you change the time range in the dashboard, then the time range in the report changes as well.
 
-## Setup
+## Requirements
 
-> SMTP must be configured for reports to be sent
+* SMTP must be configured for reports to be sent. Refer to [SMTP]({{< relref "../installation/configuration.md#smtp" >}}) in [Configuration]({{< relref "../installation/configuration.md" >}}) for more information.
+* The Image Renderer plugin must be installed or the remote rendering service must be set up. Refer to [Image rendering]({{< relref "../administration/image_rendering.md" >}}) for more information.
 
+## Create or update a report
 
-### Rendering
+Currently only Organization Admins can create reports.
 
-> Reporting requires the [rendering plugin]({{< relref "../administration/image_rendering.md#grafana-image-renderer-plugin" >}}).
-
-Reporting with the built-in image rendering is not supported. We recommend installing the image renderer plugin.
-
-## Usage
+1. Click on the reports icon in the side menu. The Reports tab allow you to view, create, and update your reports.
+1. Enter report information. All fields are required unless otherwise indicated.
+   * **Name -** Name of the report as you want it to appear in the Reports list.
+   * **Choose dashboard -** Select the dashboard to generate the report from.
+   * **Recipients -** Enter the emails of the people or teams that you want to receive the report.
+   * **Reply to -** (optional) The address that will appear in the **Reply to** field of the email. 
+   * **Custom message -** Message body in the email with the report.
```

Comment:
@xlson is the message optional or required when creating a report?

Reply:
Message and Reply To are optional

Reply:
I will update that in the doc.